includes/class-wc-payments.php

```diff
@@ -561,6 +559,16 @@ public static function init() {
 
 		self::maybe_display_express_checkout_buttons();
 
+		// Insert the Stripe Payment Messaging Element only if there is at least one BNPL method enabled.
+		$activated_bnpl_payment_methods = array_intersect(
+			Payment_Method::BNPL_METHODS,
+			self::get_gateway()->get_upe_enabled_payment_method_ids()
+		);
+
```

Comment:
To keep things logically bundled, it's better to drop this empty line.

Reply:
Sure. I have removed it here - 8275876